make develop fails when using release zip/tar.gz

```
[genadi@localhost www]$ tar -zxf 0.3.0.tar.gz
[genadi@localhost www]$ virtualenv lemur-0.3.0
New python executable in /home/genadi/Documents/www/lemur-0.3.0/bin/python
Installing setuptools, pip, wheel...done.
[genadi@localhost www]$ source lemur-0.3.0/bin/activate
(lemur-0.3.0) [genadi@localhost www]$ cd lemur-0.3.0/
(lemur-0.3.0) [genadi@localhost lemur-0.3.0]$ make develop
--> Installing git hooks
git config branch.autosetuprebase always
fatal: not in a git directory
Makefile:28: recipe for target 'setup-git' failed
make: *** [setup-git] Error 128
```

It fails because ```develop``` make target depends on ```setup-git``` target , while ```.git``` directory does not exists inside tar.gz release.

```
develop: update-submodules setup-git
        @echo "--> Installing dependencies"
```

```
setup-git:
        @echo "--> Installing git hooks"
        git config branch.autosetuprebase always
        cd .git/hooks && ln -sf ../../hooks/* ./
        @echo ""
```

Putting setup-git under comment solves the problem.

```
develop: update-submodules #setup-git
```

Or ```release``` make target should be added?

--- a/Makefile
+++ b/Makefile
@@ -20,6 +20,21 @@ endif
 	node_modules/.bin/gulp package
 	@echo ""
 
+release:
+	@echo "--> Installing dependencies"
+ifeq ($(USER), root)
+	@echo "WARNING: It looks like you are installing Lemur as root. This is not generally advised."
+	npm install --unsafe-perm
+else
+	npm install
+endif
+	pip install "setuptools>=0.9.8"
+	# order matters here, base package must install first
+	pip install -e .
+	node_modules/.bin/gulp build
+	node_modules/.bin/gulp package
+	@echo ""
+
 dev-docs:
 	pip install -r docs/requirements.txt
 
@@ -89,4 +104,4 @@ coverage: develop
 publish:
 	python setup.py sdist bdist_wheel upload
 
-.PHONY: develop dev-postgres dev-docs setup-git build clean update-submodules test testloop test-cli test-js test-python lint lint-python lint-js coverage publish
+.PHONY: develop dev-postgres dev-docs setup-git build clean update-submodules test testloop test-cli test-js test-python lint lint-python lint-js coverage publish release

